src/main/java/org/broadinstitute/hellbender/tools/exome/segmentation/AlleleFractionSegmenter.java

```diff
@@ -71,23 +73,23 @@ protected void relearnAdditionalParameters(final ExpectationStep eStep) {
         };
 
         final Function<Double, Double> meanBiasObjective = mean -> emissionLogLikelihood.apply(biasParameters.copyWithNewMeanBias(mean));
-        final double newMeanBias = OptimizationUtils.quickArgmax(meanBiasObjective, 0, AlleleFractionInitializer.MAX_REASONABLE_MEAN_BIAS, biasParameters.getMeanBias());
+        final double newMeanBias = OptimizationUtils.argmax(meanBiasObjective, 0, AlleleFractionInitializer.MAX_REASONABLE_MEAN_BIAS, biasParameters.getMeanBias(), 0.01, 0.01, 100);
```

Comment:
not sure if you want to extract these tolerances/max evals as constants, will leave it up to you